README.md

```diff
@@ -1,2 +1,91 @@
 # test-automation
-Tools for making our tests easier to run
+
```

Comment:
Hey Brian,

I think this looks pretty good now. This is a conditional :shipit: , once you fix the below items, feel free to merge to the master:

- [ ] Make sure to squash into a single (or a few) commits and have a meaningful commit message.
- [ ] Citus installation that are pre 6.0 does not have `master_add_node()`. So, for instance `tpch` tests do not work with them. Could we add workers to `pg_worker_list.conf` for pre-6.0 citus installations? (Well, you integrated that change to Citus from MX, so I think you could remember how to do pre 6 installations😄   ) I think there is no harm of adding pg_worker_list.conf for citus >= 6 installations, so maybe add to all installations to make it simple?
- [ ] tpch scale factor does not work:
```
ec2-user=# \q
[ec2-user@ip-10-192-0-174 ~]$ fab add.tpch:scale_factor=1
[localhost] Executing task 'add.tpch'
[localhost] run: readlink /home/ec2-user/pg-latest
[localhost] out: /home/ec2-user/pg-950-citus-522
[localhost] out: 

[localhost] run: make
[localhost] out: chmod 755 update_release.sh
[localhost] out: ./update_release.sh 2 13 0
[localhost] out: 

[localhost] run: SCALE_FACTOR=10 CHUNKS="o 24 c 4 P 1 S 4 s 1" sh generate2.sh
```

- [ ] Well, multiple installation capability is really neat. But, using it is not very intuitive. Please add your thoughts to #30 on how could we make it more usable? If you don't have any, please feel free to skip this.

- [ ] I couldn't see regression tests for postgres. I'm OK with not implementing/fixing this with the condition that we open an issue for that 😄 

- [ ] I tried to set preload_shared_libraries config, which I failed `fab pg.set_config:shared_preload_libraries,'citus, cstore_fdw'`. Is there a way to run this? If so, could you document it? If not, could you open an issue for that?
- [ ] In the `Ordering of fab tasks`, could you please give an example such that we could build both asserts and debug mode enabled? I mean, it is great that we have the explanation of what to do, but, I'd also want to simply copy&paste a working command with 
- [ ] Running `fab add.cstore` does not add `cstore_fdw` to preload shared libraries until `fab pg.restart`. Could we make that step automatic? 
- [ ] Due to some mistake or intentional changes, I got stuck such that `fab pg.stop` or `fab pg.restart` was erroring out saying that are you sure postgres is running. I'd really want to see that when I say `fab pg.restart`, the instances are restarted irrespective of being stopped. Believe me, I hit this quite a few times on `multiple installation` setup. And, I'm pretty sure that people will hit this a lot.